docs/cli-arguments.md

```diff
@@ -15,6 +15,7 @@
 | `--apiserver-qps` | `int` | The Kubernetes API RateLimiter maximum queries per second. | `100` |
 | `--cache-sync-timeout` | `duration` | The time limit set to wait for syncing controllers' caches. Set to 0 to use default from controller-runtime. | `2m0s` |
 | `--cluster-domain` | `string` | The cluster domain. This is used e.g. in generating addresses for upstream services. |  |
+| `--configmap-label-selector` | `string` | Limits the configmaps ingested to those having this label set to "true". If not specified, all configmaps are ingested. |  |
```

Comment:
👍 Since we're introducing this flag (no tracking of `ConfigMap`s has been done before) I believe this is the perfect timing for setting a default for this flag so that we don't start ingesting **all** the `ConfigMap`s with 3.4. 

WDYT? I'm happy with any sensible default for this as long as it's added as part of this PR. Even `konghq.com/configmap` would do but I'm open to suggestions.

Reply:
Yes, I think it makes a lot of sense. I've implemented it in ed7747580ece30f0a66e8bfd4f8d1b8016175a84